apps/condo/domains/organization/tasks/syncSbbolSubscriptionPaymentRequestsState.js

```diff
@@ -1,15 +0,0 @@
-const dayjs = require('dayjs')
-const { createCronTask } = require('@core/keystone/tasks')
-const { syncSbbolSubscriptionPaymentRequestsState } = require('@condo/domains/organization/integrations/sbbol/sync/syncSbbolSubscriptionPaymentRequestsState')
-const { getSchemaCtx } = require('@core/keystone/schema')
-
-/**
- * Syncs new and cancelled subscriptions
- */
-const task = createCronTask('syncSbbolSubscriptionPaymentRequestsState', '10 9 * * *', async () => {
```

Comment:
It's not so simple to just delete job! You already may have it inside a queue. It's better to just change it to `return true`. And remove it after some time.